buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -101,6 +100,15 @@ public void apply(Project project) {
         }
     }
 
+    @Input
+    public boolean dockerComposeSupported(Project project) {
+        // Don't look for docker-compose on the PATH yet that would pick up on Windows as well
```

Comment:
Maybe be more explicit and if (Windows) return false else return docker-compose exists?

Reply:
It's technically possible to support windows, it's just a matter of testing. 
I think being specific about the OS would make it seem like it's impossible to support.

Reply:
I don't agree, it would mean it's not supported for now. You've done that, just indirectly.

Reply:
ok, I'll add this to #36037